src/components/Header/NavLinks.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import PropTypes from "prop-types";
+import NavTitle from "./NavTitle";
+
+function NavLinks(props) {
+  const headerLinks = props.headerLinks;
+  const linksItems = headerLinks.map((item) => {
+    return <NavTitle key={item.title} title={item.title} items={item.items} />;
+  });
+
+  return <ul className={props.classForNav}>{linksItems}</ul>;
```

Comment:
Common practice is write list of components with `map`  inside return
```suggestion
  return <ul className={props.classForNav}>{ headerLinks.map((item) => 
   (  <NavTitle key={item.title} title={item.title} items={item.items} />)
   )}</ul>;
```
There are cases when it makes sence to extract components in separate variable in the same component when you have a lot of code (e.g. `src/components/Header/Dropdown.jsx` file), but not in this case